lib/filter/filter-throttle.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 Balabit
```

Comment:
For newly added files, we use `One Identity`. Could you kindly change it in this file, and the `.c` file? Thanks!
```
* Copyright (c) 2021 One Identity
```